src/__tests__/testScreen.py

```diff
@@ -36,7 +36,7 @@
     'name': 'selectTwoCommandMode',
     'input': 'absoluteGitDiff.txt',
     'inputs': ['f', 'j', 'f', 'c'],
-    'pastScreen': 1
+    'pastScreen': 3
```

Comment:
Ah cool! good to know, yeah we can capture more intermediate states this way :+1: